screenpy/questions/element.py

```diff
@@ -0,0 +1,42 @@
+"""
+A question to retrieve the element from the page, for further validation.
+Questions must be asked with an expected resolution, like so:
+
+    the_actor.should_see_the((Element(WELCOME_BANNER), IsVisible()))
+"""
+
+
+from selenium.webdriver.remote.webelement import WebElement
+
+from ..actor import Actor
+from ..pacing import beat
+from ..target import Target
+from .base_question import BaseQuestion
+
+
+class Element(BaseQuestion):
+    """
+    Ask to retrieve a specific element, viewed by an Actor. This question is
+    meant to be instantiated with a target. A typical invocation might look
+    like:
```

Comment:
I think i like this better:

```suggestion
    Ask to retrieve a specific element. This question is meant to be 
    instantiated with a target. A typical invocation might look like:
```